src/main/java/com/minecolonies/blocks/BlockBarrel.java

```diff
@@ -149,6 +151,8 @@ public int getMetaFromState(final IBlockState state)
      *
      * @return true
      */
+    //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)